lib/taskana-core/src/main/java/pro/taskana/task/api/TaskQuery.java

```diff
@@ -536,6 +536,15 @@ public interface TaskQuery extends BaseQuery<TaskSummary, TaskQueryColumnName> {
    */
   TaskQuery orderByState(SortDirection sortDirection);
 
+  /**
+   * Add the hasComments flag to the query.
+   *
+   * @param hasComments as Boolean. If null, it won't be integrated into the statement. You have to
```

Comment:
What do you mean with "You have to set false"? It should also work when setting it to true

Reply:
i copied that text from the readEquals method that works the same way as the hasComments method. It means that if it is null it will not be automatically set to false, therefore it needs to be set to false if you want to integrate it into the statement with the value false. It does work if it is set to true. Should i rephrase it for the readEquals method too?

Reply:
Yes, thanks :)